example_data/rnaseq_star/workflow.yaml

```
@@ -0,0 +1,46 @@
+instrument_data_bams:
+    - class: File
+      path: test.bam
+    - class: File
+      path: test2.bam
+outSAMattrRGline: 
+    - ['ID:2895626107', 'PU:H3MYFBBXX5.4.GCCAAT', 'SM:H_NJ-HCC1395-HCC1395_RNA', 'LB:Pooled_RNA_2891007020-cD1-lib1', 'PL:Illumina', 'CN:WUGSC']
```

```suggestion
    - ['ID:2895626107 PU:H3MYFBBXX5.4.GCCAAT SM:H_NJ-HCC1395-HCC1395_RNA LB:Pooled_RNA_2891007020-cD1-lib1 PL:Illumina CN:WUGSC']
```
See the comment about the outSAMattrRGline field in tools/star_align.cwl to see why I suggested this